src/components/creeps/creepAction.ts

```diff
@@ -27,7 +27,9 @@ export default class CreepAction implements ICreepAction {
 
   public setCreep(creep: Creep) {
     this.creep = creep;
-    this.renewStation = Game.getObjectById<Spawn>(this.creep.memory.renew_station_id);
+    let spawn = Game.getObjectById<Spawn>(this.creep.memory.renew_station_id);
+    if (spawn == null) { return false; };
```

Comment:
* `return false;` should be on its own line, and remove the trailing semicolon outside of the curly brace, e.g.

```js
if (spawn == null) {
  return false;
}
```

* The following null check has double equals while the others have triple equals. Would be nice if we have consistent use of double/triple equals.